src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:16241/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:16241/project/issues?id=sample&branch=feature4&issues=AWDaHV9BaYHLJvAGY7-z&open=AWDaHV9BaYHLJvAGY7-z)

[ISSUE_KEY]: AWDaHV9BaYHLJvAGY7-z